scipy/optimize/_linprog_ip.py

```diff
@@ -1785,33 +785,22 @@ def _linprog_ip(
 
         Minimize:     c^T * x
 
-        Subject to:   A_ub * x <= b_ub
-                      A_eq * x == b_eq
-                      bounds[i][0] < x_i < bounds[i][1]
+        Subject to:      A * x == b
+                         0 <= x < oo
 
     Parameters
     ----------
     c : array_like
         Coefficients of the linear objective function to be minimized.
-    A_ub : array_like, optional
-        2-D array which, when matrix-multiplied by ``x``, gives the values of
-        the upper-bound inequality constraints at ``x``.
-    b_ub : array_like, optional
-        1-D array of values representing the upper-bound of each inequality
-        constraint (row) in ``A_ub``.
-    A_eq : array_like, optional
+    c0 : float
```

Comment:
Yes. I did realise the docstrings for the different linprog functions are not consistent, even when documenting the same thing. I think it will be good to standardize them.